mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/150 - staff property panel (Merge this first!)

Closes #150 

To test:
- Run introductory scenario
- Open IT Staff View
- Click on Larry
- Make sure properties appear correctly
